README.md

```diff
@@ -257,6 +257,111 @@ window.navigator.getUserMedia({
 });
 ```
 
+**List of available sources:**
+
+| Sources | Description | Browsers that supports it |
+| ------- | ----------- | --------- |
+| `"window"`  | Fetches the list of application windows | Chrome, Opera, Firefox, IE, Safari |
+| `"screen"`  | Fetches the list of display screens     | Chrome, Opera, Firefox, IE, Safari |
+| `"tab"`  | Fetches the list of browser tabs | Chrome, Opera |
+| `"browser"`  | Fetches the list of browser windows | Firefox (Requires to configure `about:config` to enable `media.getusermedia.browser.enabled`)  |
+| `"application"`  | Fetches the list of applications | Firefox |
+| `"camera"`  | Fetches the list of cameras | Firefox |
+| `["tab", "audio"]`  | Fetches `"tab"` with its audio | Chrome, Opera (Enable `constraints.audio` to retrieve tab audio) |
+| `["window", "screen"]`  | Fetches `"screen"` and `"window"` | Chrome, Opera, IE, Safari |
+| `["window", "screen", "tab"]` | Fetches `"screen"`, `"window"` and `"tab"` | Chrome, Opera |
+| `["window", "tab"]` | Fetches `"window"` and `"tab"` | Chrome, Opera |
+| `["screen", "tab"]` | Fetches `"screen"` and `"tab"` | Chrome, Opera |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.screenOrWindow`  | Same as `["window","screen"]` | IE, Safari |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.screensharingKey` | Deprecated. Same as `["window","screen"]`. | IE, Safari |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.screen`  | Same as `"screen"` | IE, Safari |
+| `AdapterJS.WebRTCPlugin.plugin.screensharingKeys.window` | Deprecated. Same as `"window"`. | IE, Safari |
+
+
+## Configuring AdapterJS
+
+To configure your AdapterJS, configure this before referencing the script for `adapter.screenshare.js` or `adapter.debug.js` as an example:
+
+```
+var AdapterJS = {};
```

Comment:
@johache I understand the case, but there will be these developers and some will just not read documentation. But if that's the direction we are heading for, we have to modify the way `AdapterJS.extensionInfo` is overridden and instead of invoking the _defineScreensharingPolyfill after the end of the line, but rather do it in `webRTCReady()`. So users need not define `AdapterJS` (or for docs sake the `AdapterJS.WebRTCPlugin`) but then just override and invoke `webRTCReady()`.